### DISCUSSION TITLE
insert data into two database table from single from

### DISCUSSION THREAD

**Discussion by yasarpatel** [filamentphp/filament#6036] [General]
is there any method that we can insert data into two table.

**Comment by danharrin** [comment]
Yes, but it's a very vague question. You need to be more specific about what you're trying to achieve and where

**Reply by yasarpatel** [reply]
hi Dan,
  i have a membership form . from that membership  form there is some related table  which i want to store members  information .
 example member's children detail i want store into separate table called children.so when i register a new member it should fill all this information from single form.

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/forms/layout#saving-data-to-relationships
